content/download.html.haml

```diff
@@ -9,45 +9,92 @@
+          %ul
+            %li
+              %a{ href: "https://wiki.jenkins-ci.org/display/JENKINS/Installing+Jenkins+as+a+Windows+service", title: "Install Service"} Installing Jenkins as a Windows service
+          %p
+          %br
 
-              %p Since Jenkins was written to work on unix-like platforms, some parts assume the presence of unix-utilities. It is advised to install these as well on Windows. Install UnxUtils (this includes a shell that seems to work with forward and backwards slashes and does globbing correctly)(UnxUtils does not download), put it in the Windows PATH , and copy sh.exe to C:\bin\sh.exe (or whichever drive you use) to make shebang lines work. This should get you going. If UnxUtils gives you trouble (Fork Failed Errors), try Win-Bash.
+          %h3 Setup Wizard
+          %p Jenkins 2.0 is no longer installed completely open and accessible to the world. Instead Jenkins is initially configured to be more secure. Jenkins can no longer be accessed without a username and password and open ports are limited. During the initial run of Jenkins a security token is generated and printed in the console log:
+          %pre.code
+            *************************************************************
+            *************************************************************
+            *************************************************************
+            %p
+            Jenkins initial setup is required. A security token is required to proceed.
+            Please use the following security token to proceed to installation:
+            %p
+            41d2b60b0e4cb5bf2025d33b21cb
+            %p
+            *************************************************************
+            *************************************************************
+            *************************************************************
+          This token must be input in the Setup Wizard the first time you open the Jenkins UI. This token will also serve as the default password for the user 'admin' if you skip the user-creation step in the Setup Wizard.
+          %p
+          %p In addition to configuring the initial security for Jenkins the Setup Wizard will also install the initial plugins in your installation. Previously,Jenkins included several bundled plugins based on a set of assumed use cases. However, as technology and tools have changed the list of bundled plugins included with Jenkins has not kept up to date. Rather than continuously updating a list of bundled plugins, Jenkins 2.0 begins with a setup wizard asking the user to choose the intial plugins to be installed. The recommended set of plugins available are based on the most common use cases but users are free to customize their own installion.
+          %p
+          %br
 
-              %ul
-                %li Installing Jenkins as a Windows service
+          %h3 Case Studies
```

Comment:
Please just cut this content out, it doesn't make sense on the wiki page and it definitely doesn't make sense here.